workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -86,6 +89,23 @@ public struct Request
                 public long RequestId;
             }
 
+            /// <summary>
+            ///     Method to create a CreateEntity command request payload.
+            /// </summary>
+            /// <param name="entity">
+            ///     The Entity object that defines the SpatialOS components on the to-be-created entity.
+            /// </param>
+            /// <param name="entityId">
+            ///     (Optional) The EntityId that the to-be-created entity should take.
+            ///     This should only be provided if received as the result of a ReserveEntityIds command.
+            /// </param>
+            /// <param name="timeoutMillis">
+            ///     (Optional) The command timeout in milliseconds. Will default to the WorkerSDK default.
+            /// </param>
+            /// <param name="context">
+            ///    (Optional) A context object that will be returned to you when your receive a response.
```

Comment:
`your` -> `you`

Reply:
"When a response is received" I think we shouldn't use the second person.

Reply:
How about:

"A context object that will be returned with the command response"?